src/app/modules/project/overview/project/project.component.ts

```diff
@@ -52,8 +54,21 @@ export class ProjectComponent {
     event.stopPropagation();
   }
 
+
+  /**
+   * Method to handle the click of the like button
+   * It will either like or unlike the project
+   */
```

Comment:
I would like to suggest to add a period at the end of every sentence (I also noticed that some people won't agree on this with me, but) everywhere else on the project, comments end with a period. So, to keep everything consistent I would suggest to end this comment with a dot as well.